File: ub/pset1/mario/mario.c
#include <stdio.h>
#include <cs50.h>

void MakeBoard(int);

int main(void)
{
    int getHeight;
    //Prompt user for input
     do
     {
         getHeight = get_int("how much rows?");

     }
     while( getHeight <= 0  || getHeight > 8);

    MakeBoard(getHeight);
    return 0;
}
//create Board
 void MakeBoard(int amount)
{
    //make left side
    //get Rows
    for( int row = 0; row < amount; row++)
    {
       //create spaces
       for(int dots = amount-1; dots > row;  dots--)
        {
           printf(" ");
        }


    //createcolumn
      for(int column = 1; column <= row ; column++)
        {
              printf("#");
        }
        printf("#\n");
    }



}

File: ub/pset6/mario/less/mario.py
from cs50 import get_int

def main():
    height = GetHeight()
    row = 0
    
    while row < height:
        column = 0
        decrease = height - 1
        while decrease > row:
            print(" ",end="")
            decrease -= 1
        while column < row:
            print("#", end="")
            column += 1
        print("#")
        row += 1
    
   
        

def GetHeight():
    while True:
         height = get_int("Height: ")
         if height > 0 and height <= 8:
            break
    return height

main()


